### DISCUSSION TITLE
Ability to select where full node .chia folder is saved

### DISCUSSION THREAD

**Discussion by jonomnom** [Chia-Network/chia-blockchain#16555] [Support]
I am using Chia Wallet UI on the official website.

Is there an ability to select where .chia folder is saved because it is currently taking 121 GB on my primary drive (C:\) and I will run out of space on that drive soon. I tried installing the program in another drive, but the client keeps creating a .chia folder in the C:\ drive. 

Is there a way to configure where this folder and its contents are saved?

I really appreciate any help you can provide.

**Comment by BrandtH22** [comment]
Hey @jonomnom , what you will want to do is migrate the blockchain db to a new location.
Spacefarmers has a great guide on doing this here:
https://wiki.spacefarmers.io/guides/farming/movedb

Let us know if you have any issues migrating the db :+1:

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
